File: src/basic_training/Day24_5.java
package basic_training;

public class Day24_5 {
	public int[][] solution(int n) {
		int[][] answer = new int[n][n];
		for (int i = 0; i < n; i++) {
			for (int j = 0; j < n; j++) {
				if (i == j)
					answer[i][j] = 1;
			}
		}
		return answer;
	}
}

// 참고할 번외 문제풀이
// int[][] answer = new int[n][n];
// for(int i=0; i<n; i++) {
// 	answer[i][i] = 1;
// }
// return answer;

File: src/basic_training/Day24_5.js
function solution(n) {
	let answer = [];
	for(let i=0; i<n; i++) {
		answer[i] = Array(n).fill(0);
		answer[i][i] = 1;
	}
	return answer;
}

// 참고할만한 번외 문제풀이
// const answer = new Array(n).fill().map((_, idx)=> {
// 	const arr = new Array(n).fill(0);
// 	arr[idx] = 1;
// 	return arr;
// });

// return answer;
